assets/css/src/amp-icons.css

```diff
@@ -0,0 +1,48 @@
+#wpadminbar span.amp-icon {
+	top: 2px;
+}
+
+body .amp-icon {
+	/* stylelint-disable font-family-no-missing-generic-family-keyword */
+	font: normal 20px/1 dashicons;
+}
+
+/*
+ * Colors are taken from the official WordPress color palette.
+ * See https://make.wordpress.org/design/handbook/design-guide/foundations/colors/#auxiliary-hues
+ */
+.amp-icon.amp-valid::before {
+	content: "\f12a";
+
+	/* Accent Green */
+	color: #46b450 !important;
+}
+
+.amp-icon.amp-invalid::before {
+	content: "\f153";
+
+	/* Accent Red */
+	color: #dc3232 !important;
+}
+
+.amp-icon.amp-warning::before {
+	content: "\f534";
+
+	/* Accent Orange */
+	color: #f56e28 !important;
```

Comment:
Let's keep the color semantics we had before. Accent orange does not convey the desired semantics. This should be around `#ffff00`.

Reply:
That color might be a bit pale when in use, for example:

![image](https://user-images.githubusercontent.com/16200219/80286299-1639af00-871a-11ea-8be2-bf790aa6913e.png)

The current colors for the icons are taken from the official [WordPress color palette](https://make.wordpress.org/design/handbook/design-guide/foundations/colors/). I do agree  with the warning icon looking a bit reddish, would the Accent Yellow with 20% tint (`#ffeab3`) suffice?

![image](https://user-images.githubusercontent.com/16200219/80286375-93fdba80-871a-11ea-8cef-a12958c67d6d.png)

Reply:
That would work.